wolfssl/internal.h

```diff
@@ -2022,6 +2022,8 @@ struct WOLFSSL_CERT_MANAGER {
 #if defined(HAVE_ECC) || defined(HAVE_ED25519) || defined(HAVE_ED448)
     short           minEccKeySz;         /* minimum allowed ECC key size */
 #endif
+    wolfSSL_Mutex   refMutex;   /* reference count mutex */
```

Comment:
This was implemented because there was a double free bug when libest called `SSL_CTX_set_cert_store` but I can't replicate it now with the old implementation. I can get rid of the reference counter for this struct if you would like. This is used like other reference counters (init in `wolfSSL_CertManagerNew`, increment in `wolfSSL_CertManager_up_ref`, decrement in `wolfSSL_CertManagerFree`).